src/ansys/fluent/interface/post_objects/post_object_definitions.py

```diff
@@ -138,7 +138,7 @@ class y_axis_function(metaclass=PyLocalPropertyMeta):
         @Attribute
         def allowed_values(self):
             """Y axis function allowed values."""
-            return list(self.session.field_info.get_scalar_fields_info())
+            return list(self.session.field_data.scalar_fields())
```

Comment:
@prmukherj It appears that `field_data` and `monitors` would be the attributes here, rather than `session`. Are those the only dependencies?